client/src/pages/Contest/Contest.tsx

```diff
@@ -0,0 +1,202 @@
+      },
+      secondary: {
+        main: '#3d3d3d',
+      },
+    },
+  });
+
+  interface TabPanelProps {
+    direction?: string;
+    value: number;
+    children: React.ReactNode;
+    index: number;
+  }
+
+  useEffect(() => {
+    async function getContestAllSubmissions() {
+      try {
+        const contestAllSubmissions = await getContestById();
+        if (contestAllSubmissions) {
+          setContestSubmissions(contestAllSubmissions);
+        }
+      } catch (err) {
+        console.log(err);
+      }
+    }
+    getContestAllSubmissions();
+  }, []);
+
+  const handleChange = (event: React.ChangeEvent<Record<string, unknown>>, valueChange: number) => {
+    setValue(valueChange);
+  };
+
+  const Panel = function (props: TabPanelProps) {
+    return (
+      <div role="tabpanel" id={`${props.index}`}>
+        {value === props.index && (
+          <Container>
+            <Typography>{props.children}</Typography>
+          </Container>
+        )}
+      </div>
+    );
+  };
+
+  const capitalize = (s: string) => s && s[0].toUpperCase() + s.slice(1);
+
+  return loggedInUser ? (
+    <>
+      <CssBaseline />
+      <GlobalCss />
+      <AuthHeader linkTo="/signup" btnText="sign up" />
+      <Container className={classes.container}>
+        <Grid className={classes.grid} container style={{ marginBottom: '35px' }}>
+          {/* has to back to contests list, no route for that yet, so using dashboard for testing*/}
+          <Grid item>
+            <Typography>
+              <Link to="/dashboard" className={classes.greyText}>
+                <ArrowBackIosIcon className={classes.backIcon} />
+                Back to contests list
+              </Link>
+            </Typography>
+          </Grid>
+        </Grid>
+        <Grid className={classes.grid} container>
+          <Grid item xs={10}>
+            <Typography className={classes.contestTitle}>
+              {contestSubmissions.title ? contestSubmissions.title : 'Lion tattoo concept in minimal style'}{' '}
+              <Button className={classes.prizeAmount}>
+                <Typography className={classes.prize}>
+                  {contestSubmissions.prizeAmount ? contestSubmissions.prizeAmount : '$150'}
+                </Typography>
+              </Button>
+            </Typography>
+            <Grid direction="row" className={classes.grid} container>
+              <Grid item>
+                <Avatar alt="Profile Image" src={ProfilePic} className={classes.avatar}></Avatar>
+              </Grid>
+              <Grid item>
+                <Typography className={classes.user}>By {capitalize(loggedInUser.username)}</Typography>
+              </Grid>
+            </Grid>
+          </Grid>
+          <Grid item xs={2}>
+            <Link to={`/submit-design/${contestSubmissions.id}`} style={{ textDecoration: 'none' }}>
+              <Button variant="outlined" size="large" className={classes.button}>
+                SUBMIT DESIGN
```

Comment:
Try to avoid using capital case letter as much as you can, some screen readers pick them up as acronyms.
Please use a class to style instead.

Reply:
fixed it by using css class